
# HG changeset patch
# User hans@rhel5-devel.localdomain
# Date 1258548847 -3600
# Node ID c6549206232b21ebc4baca5476fbd3a99d9f3524
# Parent  d253187ed6cbc2e2d38c0a486a81f1fafb255049
ibv4l: update upside down devices table

From: Hans de Goede <hdegoede@redhat.com>

libv4l: update upside down devices table

Priority: normal

Signed-off-by: Hans de Goede <hdegoede@redhat.com>

--- a/v4l2-apps/libv4l/libv4lconvert/control/libv4lcontrol.c	Tue Nov 17 12:58:22 2009 +0100
+++ b/v4l2-apps/libv4l/libv4lconvert/control/libv4lcontrol.c	Wed Nov 18 13:54:07 2009 +0100
@@ -87,6 +87,8 @@
     V4LCONTROL_HFLIPPED|V4LCONTROL_VFLIPPED },
   { 0x04f2, 0xb012, 0, "PEGATRON CORPORATION         ", "X71SL     ",
     V4LCONTROL_HFLIPPED|V4LCONTROL_VFLIPPED },
+  { 0x04f2, 0xb012, 0, "PEGATRON CORPORATION", "X71TL     ",
+    V4LCONTROL_HFLIPPED|V4LCONTROL_VFLIPPED },
   /* These 3 PACKARD BELL's seem to be Asus notebook in disguise */
   { 0x04f2, 0xb012, 0, "Packard Bell BV", "T32A      ",
     V4LCONTROL_HFLIPPED|V4LCONTROL_VFLIPPED },

